wof/WaterML_1_1.py

```diff
@@ -4986,8 +4986,8 @@ def export(self, outfile, level, namespace_='', name_='siteCodeType', namespaced
         self.exportAttributes(outfile, level, already_processed, namespace_, name_='siteCodeType')
         if self.hasContent_():
             outfile.write(u'>')
-            outfile.write(u''.format(self.valueOf_))
-            self.exportChildren(outfile, level + 1, namespace_=u'', name_=u'siteCodeType', pretty_print=pretty_print)
+            outfile.write(u'{}'.format(self.valueOf_))
```

Comment:
@ocefpaf I had to fix this from your last commit since `siteCode` value was not showing up within the xml.

Reply:
We should always try to make these into tests if possible to avoid regressions.

Thanks for fixing it!